cmd/scan/scan.go

```diff
@@ -1,20 +1,27 @@
 package scan
 
 import (
+	"github.com/aws/aws-sdk-go/aws/endpoints"
 	"github.com/cloudwaste/cloudwaste/pkg/aws"
+	"github.com/cloudwaste/cloudwaste/pkg/aws/util"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	"go.uber.org/zap"
 )
 
 // Cmd runs the scan command
 func Cmd(log *zap.SugaredLogger) *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "scan",
 		Short: "Scan your cloud accounts for unused resources",
 		Run: func(_ *cobra.Command, _ []string) {
 			main(log)
 		},
 	}
+	cmd.PersistentFlags().String(util.FlagRegion, endpoints.UsEast1RegionID, "The AWS region you wish to scan")
+	viper.BindPFlag(util.FlagRegion, cmd.PersistentFlags().Lookup(util.FlagRegion))
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `viper.BindPFlag` is not checked (errcheck)